### ISSUE TITLE
Proxy request fails 

### ISSUE THREAD

**Issue by b12k** [axios/axios#1877]
#### Requests through proxy
Greetings!
I have a problem running axios with configured proxy.
I'm creating a tool to validate proxy lists.
To check if proxy is active I use its credentials as
```
axios.get(SOME_URL, {
  proxy: {
    host: proxy.host, // 81.162.195.121
    post: proxy.port // 3128
  }
})
```
And i receive either `timeout error` (thats OK) or `write EPROTO 14216:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:openssl\ssl\record\ssl3_record.c:252:`

I assume that this proxy is alive but something goes wrong.
Would be thankful for any hints!

Thank you in advance!

PS: Proxies are actually alive. Checked with `request` lib.

#### Context
- axios version: *0.18.0*
- Environment: *node v10.12.0, windows 10*

**Comment by b12k** [comment]
This issue is solved in Axios v0.19.0-beta.1.
Thanx.

**Comment by cuiyongjian** [comment]
i got the same issue.  axios throw the error although i set `proxy:false`.  But the npm package `request` is ok.
